node-fs-writefile/cp.js

```diff
@@ -0,0 +1,15 @@
+import { readFile, writeFile } from 'node:fs/promises';
+async function copy() {
+  try {
+    const name = process.argv.slice(2);
+    const newFile = name.slice(1);
+    const fileTxt = await readFile(`${name}`, 'utf8');
+    const promise = await writeFile(`${newFile}`, `${fileTxt}`);
```

Comment:
When you string interpolate `fileTxt` you convert it to a string. This is fine for text files but will ruin the contents of an image file.